### ISSUE TITLE
Add Claude and Google models into benchmark

### ISSUE THREAD

**Issue by rodion-m** [booydar/babilong#3]
Thanks a lot a such uniq benchmark! Could you please add great models from Claude to the benchmark: Haiku, Sonnet and Opus?
And, Google Gemini Pro and Gemini Flash also.

**Comment by booydar** [comment]
Thanks for your interest in our work! Currently we had time and resources to evaluate only some of the main models in the field. We'll try to extend the model selection in the future, and any help from the community is highly appreciated! New models can be added to the leaderboard via pull request of evaluation results to this repository.

**Comment by rodion-m** [comment]
Yes, thanks so much for all your hard work. I just wanted to mention that the Claude and Google models are looking great at the moment. And the new Llama 405b is pretty impressive too. When you get a chance, I'd love to see your benchmarks for all of them. Thanks again!

**Comment by yurakuratov** [comment]
We have just added results for LLama-3.1-Instruct (8B and 70B). LLama-3.1 70B shows very strong results on long contexts (32k+), even outperforming GPT-4.

![image](https://github.com/user-attachments/assets/c76143d4-d18c-47fb-8ce4-c4f145f20a08)

**Comment by rodion-m** [comment]
It's great! Thank you.

May I ask what prevents you from adding Claude, Gemini and llama 405b to the benchmark?

If you wish, I can assist with providing an OpenRouter api key for that purpose. You can contact me via telegram: https://t.me/rodion_m_tg
@yurakuratov

**Comment by yurakuratov** [comment]
We have evaluated Gemini 1.5 Pro 002 on BABILong tasks:
<img width="876" alt="image" src="https://github.com/user-attachments/assets/e7939252-c77d-489c-a663-d3673f1289d2">
<img width="876" alt="image" src="https://github.com/user-attachments/assets/78586862-740c-47be-aa33-e1cafdf8dc2c">
It is currently the strongest LLM we have evaluated.

These results are included in updated [paper](https://arxiv.org/abs/2406.10149v2).
